src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_SIZE = 20;
+
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();

```

Comment:
Consider declaring 'random' and 'supplier' as final since they are not intended to be changed after initialization.